### ISSUE TITLE
Set same styling on all series

### ISSUE THREAD

**Issue by vincentmacri** [metabase/metabase#10507]
**Is your feature request related to a problem? Please describe.**
For a line chart with multiple series, I want to show the top X series with the highest value and use the curve line style while showing dots on the lines. When a new series replaces an old one in the top series, the new series has the default styling.

**Describe the solution you'd like**
On the display tab when editing a visualization, I want to have the ability to set the default styling for the series in my visualization.

**Describe alternatives you've considered**
I did not see any features that could function as an alternative.

**How important is this feature to you?**
This is quite important. It feels like a very basic feature.

**Additional context**
![MetabaseFeatureRequest1](https://user-images.githubusercontent.com/9146157/62572689-5098ae80-b862-11e9-8061-4b8f75ddc794.png)
I want all series to have the same styling I set here for "Old Series 3".

![MetabaseFeatureRequest2](https://user-images.githubusercontent.com/9146157/62572712-5d1d0700-b862-11e9-9dcf-5983f67b5c85.png)
"New Series" has the default styling, and there is no way for me to change the default styling per-visualization (or even system-wide).

**Comment by lindsay-stevens** [comment]
Not being able to set default series display settings makes it tedious to manually change the settings for all series when there are more than a couple of series.

Similarly, it makes it unnecessarily complicated to programmatically create questions with a certain style. That is, in order to set the same style on all series, the question (or something equivalent) needs to be executed to get the names of the series, then using those series names set the display settings.

I'm sure there are edge/use cases for being able to set different styles for each series but I would imagine that (like me), most of the time, users want the same style on all series. For example with the Line visualisation, would not alternate between straight/smooth/stepped line, interpolation, or dots, depending on the series.

**Comment by MartinCura** [comment]
Just FYI, this is still an issue, right? I'd love to change the default styling.

**Comment by csyperski** [comment]
This has been an issue for us as well as our series are dynamical and are not known at the time of chart creation, like  a student at a school, but there is no way to know ahead of time the student number that will be used as they may only display on the chart when something is triggered like a low gpa, so we can't stylize the x-axis in a way that makes sense, like replacing missing values with Zero.  It would be awesome to have a global for each line chart!

**Comment by wajdifarid** [comment]
> This has been an issue for us as well as our series are dynamical and are not known at the time of chart creation, like a student at a school, but there is no way to know ahead of time the student number that will be used as they may only display on the chart when something is triggered like a low gpa, so we can't stylize the x-axis in a way that makes sense, like replacing missing values with Zero. It would be awesome to have a global for each line chart!

I agree. I am also facing the same issue. This is small but impactful

**Comment by nepomuc** [comment]
@cdeweyx Any updated on when this will be available?

**Comment by Tony-metabase** [comment]
In recent versions we added more options:

<img width="1485" height="699" alt="Image" src="https://github.com/user-attachments/assets/bc6c65c2-f415-4300-8acd-f655dea68488" />

Which makes this worse to individually go over each non deterministic label (since you will be filtering)

**Comment by Batwam** [comment]
I'm having a similar issue. I have plots with 30 lines and setting each of them to small is impractical
